### PR TITLE
feat: If only 1 shopping list, navigate straight to it (Shopping List QoL)

### PR DESCRIPTION
**Pull request by boc-the-git**
## What type of PR is this?

- feature

## What this PR does / why we need it:

If you only have one shopping list available to you at `/shopping-lists`, this will navigate you directly into it. There is also a button to go back to the list that disables that behaviour (via a query param) should you wish.

shopping_list_redirect

## Which issue(s) this PR fixes:

None.

## Special notes for your reviewer:

Credit @Kuchenpirat for the v-container stuff on _id.vue <3

## Testing

Manual

### CODE REVIEW

File: frontend/pages/shopping-lists/_id.vue [hunk 1]
```diff
@@ -2,7 +2,16 @@
   <v-container v-if="shoppingList" class="md-container">
     <BasePageTitle divider>
       <template #header>
-        <v-img max-height="100" max-width="100" :src="require('~/static/svgs/shopping-cart.svg')"></v-img>
+        <v-container>
+          <v-row>
+            <v-col cols="3" class="text-left">
+              <ButtonLink :to="`/shopping-lists?disableRedirect=true`" :text="$tc('shopping-list.back-to-lists')" :icon="$globals.icons.backArrow" />
```
Comment (Kuchenpirat) [inline frontend/pages/shopping-lists/_id.vue]: We already have a back
```suggestion
              <ButtonLink :to="`/shopping-lists?disableRedirect=true`" :text="$tc('general.back')" :icon="$globals.icons.backArrow" />
```

File: frontend/lang/messages/en-US.json [hunk 2]
```diff
@@ -784,6 +784,7 @@
   },
   "shopping-list": {
     "all-lists": "All Lists",
+    "back-to-lists": "Back",
```
Comment (Kuchenpirat) [inline frontend/lang/messages/en-US.json]: not needed
```suggestion
```

**Review by Kuchenpirat — CHANGES_REQUESTED** [review]
looks good, just an unnecessairy translation and were good to go

**Comment by boc-the-git** [comment]
Good call @Kuchenpirat, after the wording change the new translation wasn't necessarily! 🙈

**Review by Kuchenpirat — APPROVED** [review]
All good now. 

Exited for this 🚀  Usually have only one list and this saves one step a lot of times 😊

**Pull request merged**